src/uwtools/tests/test_logging.py

```diff
@@ -0,0 +1,52 @@
+# pylint: disable=missing-function-docstring
+"""
+Tests for uwtools.logging module.
+"""
+
+import logging
+import os
+from unittest.mock import ANY, patch
+
+from pytest import raises
+
+import uwtools.logging
+
+# NB: Since pytest takes control of logging, these tests make assertions about the way logging
+# setup calls were made instead of obtaining the root logger object and making assertions about
+# it, since the roog logger will not be what one would expect.
+
+
+def test_setup_logging():
+    with patch.object(logging, "basicConfig") as basicConfig:
+        uwtools.logging.setup_logging()
+        basicConfig.assert_called_once_with(
+            datefmt=ANY,
+            format=ANY,
+            level=logging.INFO,
```

Comment:
Use the `ANY` value for `datefmt` and `format` to avoid getting to specifc about these, as their values are not really what we're testing for here. (We're testing that `filename=` was not specified, since we're not in `--quiet` mode; and that the level is correct/default.)